packages/@aws-cdk/aws-s3/test/bucket.test.ts

```diff
@@ -724,7 +724,19 @@ describe('bucket', () => {
         bucketName: 'mybucket',
       });
 
-      expect(bucket.bucketWebsiteUrl).toEqual(`http://mybucket.s3-website.us-east-2.${stack.urlSuffix}`);
+      expect(bucket.bucketWebsiteUrl).toEqual('http://mybucket.s3-website.us-east-2.amazonaws.com');
+    });
+
+    test('old bucketWebsiteUrl format for specific region', () => {
+      const stack = new cdk.Stack(undefined, undefined, {
+        env: { region: 'us-east-1' },
```

Comment:
Let's add a test for a China region to check we get the correct urlSuffix

Reply:
Sure, nice idea 👌